### DISCUSSION TITLE
iframe mode footer problem

### DISCUSSION THREAD

**Discussion by ozkandanaci** [ColorlibHQ/AdminLTE#3272] [Q&A] [answered]
Hi, 
the footer area does not disappear  from bottom , when remove tag of `<footer class="main-footer">`  in iframe mode . there is an empty place always in footer place as placeholder.  how can i remove this place 
version : AdminLTE v3.1.0-rc

![Captur](https://user-images.githubusercontent.com/51021423/103438815-5dccef00-4c48-11eb-9da0-159ead6694b2.PNG)

**Comment by REJack** [comment, marked as answer]
I've fixed this problem with e4ab1e4737537b4a0fbcad2c00bfab199d0c7c05, please create the next time for issues/problems a issue instead of a discussion 😄
